ClassModules/Druid.lua

```diff
@@ -3403,6 +3422,8 @@ barContainerFrame:SetScript("OnEvent", function(self, event, ...)
 					end
 				elseif entry.spellId == spells.starfall.id then
 					snapshotData.snapshots[entry.spellId].buff:Initialize(entry.type)
+				elseif entry.spellId == spells.starlord.buffId then
+					snapshotData.snapshots[spells.starlord.id].buff:Initialize(entry.type)
```

Comment:
When changing the spell definition to use `id` and `talentId`:

```suggestion
				elseif entry.spellId == spells.starlord.id then
					snapshotData.snapshots[spells.starlord.id].buff:Initialize(entry.type)
```

Reply:
Also I don't think Dream Burst will be as complicated, since that (should) be a straight forward grabbing of the buff's time and stacks. This one is the more complicated/nuanced talent between the two IMO.

Extra complexity for either would be if you wanted, say, a border color change given some parameters. Examples I can think of:
- Starlord - border color change when you're at 3 stacks (or maybe not?), with logic to do checking if the talent is enabled.
- Dream Burst - border color change when you have charges available, similar to Mind Flay: Insanity for Shadow.